infrastructure/cli/commands/s3/put.ts

```diff
@@ -33,14 +33,13 @@ interface Options extends EnvironmentOptions {
 export async function handler({environment, key, openFrom}: Options): Promise<void> {
   const client = await Client.load(environment)
   // eslint-disable-next-line init-declarations
-  let content
+  let readableContent: Readable
   if (openFrom) {
     const openFromPath = path.resolve(openFrom)
     const readHandle = await fs.open(openFromPath, 'r')
```

Comment:
you shouldn't need the `readHandle`/`await fs.open(openFromPath, 'r')` anymore now that you just assign a readStream to `readableContent` right?

Reply:
i think i have too strong an obsession with choosing the async/promises form of standard library methods ;)

Reply:
I like the async/promises versions for most things as well :)